src/js/views/patients/sidebar/action/action-sidebar_views.js

```diff
@@ -307,6 +392,9 @@ const LayoutView = View.extend({
   showDisabledSave() {
     this.showChildView('save', new DisabledSaveView());
   },
+  showCommentForm() {
+    this.showChildView('comment', new CommentFormView({ actionId: this.model.id }));
```

Comment:
I think we'll probably want to modify this form in the future to handle not only new comments but edit them as well.  It may be best to pass in the comment model.   Really I think it might be best to pull showing the comment view out to the app level and have the app construct the data.  The app will also probably need to eventually add the successfully posted comment to the comment collection so should probably handle the `saveAll` as well.